offchain-modules/lib/src/transfer/to_eth.rs

```diff
@@ -29,14 +48,96 @@ pub fn burn(private_key: String, rpc_url: String) -> Result<String> {
         "tx_hash": hex::encode(tx.hash().as_slice()),
         "cell_typescript": cell_script,
     });
-    println!("{}", serde_json::to_string_pretty(&print_res).unwrap());
+    debug!("{}", serde_json::to_string_pretty(&print_res).unwrap());
     Ok(hex::encode(tx.hash().as_slice()))
 }
 
-pub fn parse_ckb_proof() -> Result<()> {
+pub fn unlock() -> Result<()> {
     todo!()
 }
 
-pub fn unlock() -> Result<()> {
-    todo!()
+pub fn get_add_ckb_headers_abi(headers: Vec<u8>) -> Bytes {
+    //TODO : addHeader is mock function for test feature which set header data in eth contract
+    let function = Function {
+        name: "addHeader".to_owned(),
+        inputs: vec![Param {
+            name: "_data".to_owned(),
+            kind: ParamType::Bytes,
+        }],
+        outputs: vec![],
+        constant: false,
+    };
+    function.encode_input(&[Token::Bytes(headers)]).unwrap()
+}
+
+pub fn parse_ckb_proof(tx_hash_str: &str, rpc_url: String) -> Result<CkbTxProof, String> {
+    let tx_hash = covert_to_h256(tx_hash_str)?;
+    let mut rpc_client = HttpRpcClient::new(rpc_url);
+    let retrieved_block_hash;
+    let retrieved_block;
+    let mut tx_indices = HashSet::new();
+    let tx_index;
+    match rpc_client.get_transaction(tx_hash.clone())? {
+        Some(tx_with_status) => {
+            retrieved_block_hash = tx_with_status.tx_status.block_hash;
+            retrieved_block = rpc_client
+                .get_block(retrieved_block_hash.clone().expect("tx_block_hash is none"))?
+                .expect("block is none");
+
+            tx_index = get_tx_index(&tx_hash, &retrieved_block)
+                .expect("tx_hash not in retrieved_block") as u32;
+            dbg!(tx_index);
```

Comment:
remove `dbg!` after your debug. Or turn it into `debug!`.

Reply:
> remove `dbg!` after your debug. Or turn it into `debug!`.

done